cloudferry/lib/os/storage/cinder_storage.py

```diff
@@ -341,6 +345,11 @@ def create_volume(self, size, **kwargs):
         cinder = self.cinder_client
         tenant_id = kwargs.get('project_id')
 
+        if self.cloud.position == 'dst':
```

Comment:
I think `volume_type` should be treated equally for both source and destination, I'd suggest removing this check